lua/astrocommunity/completion/avante-nvim/init.lua

```diff
@@ -47,6 +51,25 @@ return {
   },
   specs = { -- configure optional plugins
     { "AstroNvim/astroui", opts = { icons = { Avante = "" } } },
+    {
+        "saghen/blink.cmp",
+        dependencies = {
+          "Kaiser-Yang/blink-cmp-avante",
+        },
+        opts = function(_, opts)
+          opts.sources = vim.tbl_deep_extend("keep", opts.sources or {}, {
+            default = {},
+            providers = {
+              avante = {
+
+                module = "blink-cmp-avante",
+                name = "Avante",
+              },
+            },
+          })
+          table.insert(opts.sources.default, "avante")
+        end,
+      },
```

Comment:
```suggestion
      {
        "Kaiser-Yang/blink-cmp-avante",
        lazy = true,
        specs = {
          {
            "Saghen/blink.cmp",
            optional = true,
            opts = {
              sources = {
                default = { "avante" },
                providers = {
                  avante = { module = "blink-cmp-avante", name = "Avante" },
                },
              },
            },
          },
        },
      },
```

this should be simplified to this